ote/resources/public/language/sv.edn

```diff
@@ -540,6 +540,9 @@
                :hours    "Tuntia"
                :days     "Päivää"
                :seconds  "Sekuntia"}
+  :save-failed                                "Tallennettaessa sattui virhe. Tarkista lomakkeen tiedot"
```

Comment:
Phrasing blames the user, perhaps we need a link to open feedback form when an error is shown.